pings.yaml

```diff
@@ -9,6 +9,18 @@
 ---
 $schema: moz://mozilla.org/schemas/glean/pings/2-0-0
 
+study-enrollment:
+  description: |
+    This ping is sent when the user has consented to the study.
+  include_client_id: false
+  send_if_empty: true
+  bugs:
+    - https://github.com/mozilla-rally/rally-study-01/issues/114
+  data_reviews:
+    - https://github.com/mozilla-rally/rally-core-addon/pull/505#issuecomment-815826426
```

Comment:
This data-review seems unrelated to this ping. Is [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1663857#c5) maybe the right one?